packages/components/psammead-episode-list/src/Link.jsx

```diff
@@ -1,8 +1,27 @@
 import styled from '@emotion/styled';
-import { C_METAL, C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
+import {
+  C_METAL,
+  C_POSTBOX,
+  C_WHITE,
+  C_STONE,
+} from '@bbc/psammead-styles/colours';
 
 const Link = styled.a`
-  display: block;
+  :before {
+    position: absolute;
+    inset: 0;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    content: '';
+    white-space: nowrap;
```

Comment:
is this needed?

Reply:
Doesn't seem to have any effect - removed 👍